proto/distributed-chat.proto

```diff
@@ -24,7 +24,7 @@ message ChatRequest {
 message Citation {
   string id = 1;
   string title = 2;
-  string uri = 3;
+  repeated string uris = 3;
```

Comment:
This is also used in inference, so wouldn't that need to change to match?

Reply:
it is already changed in inference, not sure why it wasn't changed here

Reply:
Probably because it was changed in PLEX inference, not here. But what I was really asking is did inference get a change to send multiple uris in a single citation?

Reply:
yes it got changed